CHANGELOG.md

```diff
@@ -926,7 +818,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - Run with tag 0.1.0 [`c634ad2`](https://github.com/Human-Connection/Human-Connection/commit/c634ad264bd99dd1a87a86f870d7877aa751dc38)
 - zwischenspeichern [`e4c7c11`](https://github.com/Human-Connection/Human-Connection/commit/e4c7c1125da6f8fa259241b4d3838b1e7b1e24a2)
 
-#### v0.1.0
```

Comment:
@mattwr18 could you update your `git tag`s please and re-generate the changelog, please?